Makefile

```diff
@@ -38,7 +38,9 @@ welcome:
 	@echo ""
 	@echo "Please see our Quick Start Guide at:"
 	@echo "    https://doc.riot-os.org/getting-started.html"
-	@echo "Or ask questions on our mailing list:"
+	@echo "Or ask questions on our forum"
+	@echo "    https://forum.riot-os.org"
+	@echo "or mailing list:"
```

Comment:
For consistency, I would prefer both "or"s to either start with a capital or a lower case "o". Please squash right away.

Reply:
But it's the middle of the sentence. Now that I read it myself again, I am not happy about the double "or" though... Let me rephrase.

Reply:
Done and squashed.